### ISSUE TITLE
ALMNH - Relationships

### ISSUE THREAD

**Issue by mbprondzinski** [ArctosDB/data-migration#186]
What ever became of track parts and counterparts?  I tried relating two specimens and couldn't find these names.  What are we using?  I created the relationship "collected with" but this seems too broad.  Thanks!

**Comment by campmlc** [comment]
If the parts of same individual are cataloged separately, use " same
individual as". The relationship relates to the cataloged item, not to
individual parts.

**Comment by mbprondzinski** [comment]
Teresa started a discussion awhile ago on Ichnofossil slabs which sometimes have counter(impressions)parts to them, and I wasn't sure if an agreement was reached on how to relate these or not?  Anyway, they are two distinct specimens of the same animal so I guess for now I will use "same individual as."

**Comment by campmlc** [comment]
Are they currently cataloged separately?

**Comment by mbprondzinski** [comment]
Yes, they have their own catalog numbers.

**Comment by mbprondzinski** [comment]
Interesting... Arctos doesn't like the "same individual as" relationship.  It put a big red box around the relationship!
![image](https://user-images.githubusercontent.com/43214224/72637241-77d1cc80-3926-11ea-8560-fac300380c6c.png)

**Comment by dustymc** [comment]
That's a "Hey user, you might be missing important stuff in linked specimens" red.

Also, that link only "works" (in a limited capacity) because we are doing things which break functionality with relationships (https://github.com/ArctosDB/arctos/issues/2278). Linking via a GUID - the ALMNH:ES number is the most obvious one - will make that specimen much more usable.

**Comment by campmlc** [comment]
Yes, I find the red box obnoxious too, but it certainly draws attention.
What are the the ALMNH:ES catalog numbers?

**Comment by mbprondzinski** [comment]
The records are ALMNH 6172 and 6173.

However, the reason this came up at all is because the PV numbers were erroneously cited in a 2011 publication as an Invertebrate HOLOTYPE (!)  I have yet to document/deal with that!

**Comment by campmlc** [comment]
Create the relationship "same individual as between the ALMNH:ES numbers,
which allows you to link the related items in your collection. Then you add
the PV number as an OtherID to both record, with relationship "self". The
only reason to use the "same individual as" relationship is to create a
link to another object with a url (or in some cases, to link to a duplicate
catalog number in a non-Arctos institution.)
The holotype citation should be made using the ALMNH:ES number, and people
can then find the same specimens using the PV number that is an OtherID in
both records.

**Comment by dustymc** [comment]
>obnoxious

It's just CSS - it's easy enough to control, or you can change it for your collection.

>erroneously cited

http://arctos.database.museum/info/ctDocumentation.cfm?table=CTCITATION_TYPE_STATUS&field=erroneous%20citation

The purpose of that identifier should be to link specimens in the most unambiguous way possible. Other data can be stored elsewhere.

**Comment by mbprondzinski** [comment]
Okay, I'll try to fix this next week.  Thanks all!
